elide-datastore/elide-datastore-aggregation/src/test/java/com/yahoo/elide/datastores/aggregation/AggregationDataStoreTransactionTest.java

```diff
@@ -255,6 +260,7 @@ public void loadObjectsExceptionThrownTest() throws Exception {
     }
 
     @Test
+    @MockitoSettings(strictness = Strictness.LENIENT)
```

Comment:
What does this do?

Reply:
reverted it, no longer required after after reverting the changes to base class. Earlier I was getting this error: ```Unnecessary stubbings detected.
Please remove unnecessary stubbings or use 'lenient' strictness.```